src/main/java/core/basesyntax/Solution.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+
+public class Solution {
+
+    public static void main(String[] args) {
+        ArrayList<Figure> figuresArray = new ArrayList<>();
```

Comment:
```suggestion
        List<Figure> figuresArray = new ArrayList<>();
```
хорошою практикою вважається використання більш абстрактного типу щоб зробити код більш гнучким